lisa/tools/timeout.py

```diff
@@ -11,7 +12,7 @@ def can_install(self) -> bool:
         return False
 
     def run_with_timeout(
-        self, command: str, signal: str, timeout: int, kill_timeout: int = 0
+        self, command: str, timeout: int, signal: int = SIGTERM, kill_timeout: int = 0
```

Comment:
How about put SIGINT as default? Is it softer?

Reply:
I think TERM is the 'nice' way to ask for a kill normally. Testpmd likes INT in interactive mode but it's less common (I think).